src/devices/Ssd1306/Command/ICommand.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ssd1306.Command
+{
+    public interface ICommand
+    {
+        byte Value { get; }
```

Comment:
This is not super clear, should it be sort of an ID instead?

Reply:
Yeah, that sounds reasonable.  I will update.  I just named it just like the generic GetBytes method.